src/bot/MessageHandler.ts

```diff
@@ -16,16 +16,16 @@ export default class MessageHandler {
     this.commands = commands;
   }
 
-  public handle(message: Message) {
-    if (!this.isValidMessage(message)) return;
+  public async handle(message: Message) {
+    if (!MessageHandler.isValidMessage(message)) return;
 
     const messageToHandle = message;
     messageToHandle.content = message.content.substring(config.prefix.length);
 
-    this.execute(messageToHandle);
+    await this.execute(messageToHandle);
```

Comment:
I do not think awaiting this is necessary, as we are not doing anything afterwards.